### PR TITLE
add wheel upload to anaconda sunpy channel

### PR DESCRIPTION
**Pull request by nabobalis**
I added a token to the action secrets page.

### CODE REVIEW

File: tox.ini
Comment (Cadair) [inline tox.ini:1]: Why do you think we should remove this? Surely all the reasons we added them still apply? online tests are flakey and sometimes re-running helps and sometimes they stall out forever?

Also this change isn't actually what the PR says it is about.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -184,11 +188,16 @@ jobs:
...
+      anaconda_token: ${{ secrets.anaconda_token }}
 
   publish_pure:
     needs: [publish]
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@main
     with:
+      upload_to_anaconda: ${{ (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') }}
+      anaconda_user: sunpy
+      anaconda_package: sunpy
+      anaconda_keep_n_latest: -1
```
Comment (Cadair) [inline .github/workflows/ci.yml:200]:
```suggestion
      anaconda_keep_n_latest: 10
```

File: tox.ini
Comment (nabobalis) [reply tox.ini:1]: I will remove the change and we can debate this in another PR

**Review by Cadair — APPROVED** [review]

**Comment by nabobalis** [comment]
I think this counts as a minor change.

**Pull request merged**